ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/datastructures/forkchoice/TestStoreImpl.java

```diff
@@ -257,6 +266,11 @@ public void setTime(final UInt64 time) {
     this.time = time;
   }
 
+  @Override
+  public void setTimeMillis(final UInt64 time) {
+    this.timeMillis = time;
+  }
```

Comment:
I worry that having the time stored in both seconds and millis will get out of sync and potentially be very confusing.  So I'd probably just store the time in millis internally.  `getTimeSeconds` and `setTimeSeconds` would then do the conversion between seconds and millis.

Reply:
I agree, now that i am making the next changes on top, that bit got me worried as well, will resolve it in this PR first